.github/workflows/release.yaml

```diff
@@ -0,0 +1,24 @@
+name: Release Che Machine Exec
+
+on:
+  workflow_dispatch:
+    # Inputs the workflow accepts.
+    inputs:
+      version:
+        description: 'The version that is going to be released. Should be in format 7.x.y'
+        required: true
+jobs:
+  build:
+    name: Create Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Create Release
+        run: |
+          git config --global user.name "Mykhailo Kuznietsov"
+          git config --global user.email "mkuznets@redhat.com"
+          export GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }}
+          docker login --username ${{ secrets.DOCKERHUB_USERNAME }} --password ${{ secrets.DOCKERHUB_PASSWORD }}
+          docker login quay.io --username ${{ secrets.QUAY_USERNAME }} --password ${{ secrets.QUAY_PASSWORD }}
+          /bin/bash make-release.sh --repo git@github.com:eclipse/che-machine-exec --version ${{ github.event.inputs.version }} --trigger-release
```

Comment:
In this job https://github.com/eclipse/che-machine-exec/blob/master/.github/workflows/gh_actions_push.yaml#L37 we have ability to build che-machine exec nightly for few platforms. I think, it would be nice to have such ability for released version too. What do you think?

Reply:
please open a separate issue... right now the make-release.sh script needs a version from which it can compute tags/branches. For nightly, what would be the branch/tag to use?

Reply:
>  For nightly, what would be the branch/tag to use?

Tag will be nightly:

=> https://github.com/eclipse/che-machine-exec/blob/master/.github/workflows/gh_actions_push.yaml#L39

> please open a separate issue...

OK, https://github.com/eclipse/che/issues/18509

Reply:
OK so insttead of single-platform code:

```
releaseMachineExec() {
  GIT_COMMIT_TAG=$(git rev-parse --short HEAD)
  docker build -t ${IMAGE} -f ./${DOCKERFILE} . | cat
  tag_push "${REGISTRY}/${ORGANIZATION}/${IMAGE}:${GIT_COMMIT_TAG}"
  echo "'${GIT_COMMIT_TAG}' version of images pushed to '${REGISTRY}/${ORGANIZATION}' organization"
  
  tag_push "${REGISTRY}/${ORGANIZATION}/${IMAGE}:${VERSION}"
  echo "'${VERSION}'  version of images pushed to '${REGISTRY}/${ORGANIZATION}' organization"
}
```

we'd do everything in make-release.sh EXCEPT the actual build (`releaseMachineExec()`), then call:

```
    - name: Set up QEMU
      uses: docker/setup-qemu-action@v1
    - name: Set up Docker Buildx
      uses: docker/setup-buildx-action@v1
    - name: Login to quay.io
      uses: docker/login-action@v1 
      with:
        username: ${{ secrets.QUAY_USERNAME }}
        password: ${{ secrets.QUAY_PASSWORD }}
        registry: quay.io
    - name: Build and push nightly
      uses: docker/build-push-action@v2
      with:
        file: build/dockerfiles/Dockerfile
        platforms: linux/amd64,linux/ppc64le,linux/s390x
        push: true
        tags: quay.io/eclipse/che-machine-exec:${{ github.event.inputs.version }}
```

Reply:
... but how to split the make-release.sh script into 2 pieces so we can do all the tagging and pushing and branching AND do the builds via GH action at the right time? *strokes chin, ponders*